Public/app.js

```diff
@@ -276,6 +307,61 @@ function updateHighlight(gridItems) {
     currentIndex = nextIndex;
 }
 
+// used to navigate between category and settings
+function handleSKeyPressedSettings() {
+    // chooses which setting to modify
+    if (currentSectionDisplayed === sectionList.SETTINGS && choosingSetting === "none") {
+        choosingSetting = true;
+        settingIndex = currentIndex;
+        console.log(settingIndex);
+        clearInterval(intervalID);
+        
+        if (settingIndex == 0) {
+            highlightGridItems(volumeItems);
+            choosingSetting =  volumeItems;
+            choosingSetting.forEach(item => item.classList.remove("setting-selected"));
+        }
+        else if (settingIndex == 1) {
+            highlightGridItems(hlightItems);
+            choosingSetting = hlightItems;
+            choosingSetting.forEach(item => item.classList.remove("setting-selected"));
+        }
+        else {
+            // navigate back to category screen
+            currentSectionDisplayed = sectionList.CATEGORY;
+            settingsSection.style.display = "none";
+            categorySection.style.display = "block";
+            clearInterval(intervalID);
+            highlightGridItems(categoryItems);
+            choosingSetting = "none"
+            displayCategories();
+        }
+    }
+    // chooses which value to change the setting to
+    else if (currentSectionDisplayed === sectionList.SETTINGS && choosingSetting !== "none") {
+        let index = currentIndex;
+        let setting_value;
+        choosingSetting.forEach(item => item.classList.remove("highlight-tab"));
+        setting_value = choosingSetting[index];
+        choosingSetting[index].classList.add("setting-selected");
+    
+        if (choosingSetting === volumeItems) {
+            volumePercent = setting_value.dataset.value;
```

Comment:
We want to convert the string to a number [see docs](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/volume), not sure if this works otherwise